overkilling/overkill-todo-spa-frontend
# Show simple Todo list in homepage

## Description

Closes #1 

Very simple todo list implementation, with no other functionality. 

It does not represent the best way of fetching data from an API, or how to display and style a list of items. However, it is a basic [walking skeleton](https://wiki.c2.com/?WalkingSkeleton) of a SPA with just enough functionality to count as a frontend app.

Future PRs will gradually improve the functionality, techniques, and style. Hopefully.

 ## How to test
Steps to test :
1. Start up the SPA  with an API
2. Go to `http://localhost:8080/`
3. Verify there's a Todo list
